packages/kyt-cli/cli/actions/setup.js

```diff
@@ -371,42 +459,83 @@ module.exports = (config, flags, args) => {
       ];
       inquire.prompt(question).then((answer) => {
         if (answer.srcBackup) {
-          startSetup();
+          callStarterSetup();
         } else {
           process.exit();
         }
       });
     } else {
-      startSetup();
+      callStarterSetup();
     }
   };
 
+  // setup tasks for setup in existing project
+  const existingProjectSetup = () => {
+    logger.start('Setting up kyt');
+    updateUserPackageJSON(true);
+    createEditorconfigLink();
+    createESLintFile();
+    createStylelintFile();
+    createKytConfig();
+    createGitignore();
+    logger.end('Done setting up kyt');
+  };
+
   // Selects type of setup
   const setupPrompt = () => {
-    // Skip starter-kyt question if they've already supplied a repo name
+    // Skip starter-kyt questions if they've already supplied a repo name
     if (args.repository) {
-      srcPrompt(starterKytSetup);
+      srcPrompt();
     } else {
       const question = [
         {
           type: 'confirm',
           name: 'setupStarter',
-          message: 'Would you like to setup with the default starter-kyt?',
-          default: false,
+          message: 'Would you like to setup with a starter-kyt?',
+          default: true,
         },
       ];
       inquire.prompt(question).then((answer) => {
         if (answer.setupStarter) {
-          srcPrompt(starterKytSetup);
+          srcPrompt();
+        } else {
+          existingProjectSetup();
+        }
+      });
+    }
+  };
+
+  // Checks to see if user is running current version of kyt-cli
+  // Gives option to exit if version is old
+  // runs setup
+  const checkCliVersionPrompt = () => {
+    const currentVersion = cliPkgJson.version;
+    const output = shell.exec('npm info kyt-cli version');
+    // If kyt-cli is up to date, proceed
+    // TODO: remove code check after cli is published
+    if (output.code !== 0 || !semver.lt(currentVersion, output.stdout.trim())) {
+      setupPrompt();
+    } else {
+      const question = [
+        {
+          type: 'confirm',
+          name: 'cliVersion',
+          message: 'There is a newer version of kyt-cli available. \n We recommend you upgrade before you continue. \n Would you like to proceed anyway?',
```

Comment:
could this message output the installed and latest version of kyt-cli? (e.g. "You have 0.4.0 installed, the latest is 0.5.2")